Polyfills/Canvas/Source/Font.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+#include <optional>
+
+namespace Babylon::Polyfills::Internal
+{
+    enum class FontStyle
+    {
+        Normal,
+        Italic,
+    };
+
+    struct Font
+    {
+    public:
+        operator std::string() const;
+        static std::optional<Font> Parse(const std::string& fontString);
+
+        float GetSize() const { return size; }
+        const std::string& GetFamiliy() const { return family; }
```

Comment:
nit: Spelling, though I would recommend we do it like this.

```suggestion
        float Size() const { return size; }
        const std::string& Family() const { return family; }
```

Reply:
fixed